build.gradle

```diff
@@ -12,6 +24,11 @@ plugins {
     alias libs.plugins.org.jetbrains.kotlin.jvm apply false
 }
 
+allprojects {
+    // add ktlint to all modules
+//    apply plugin: libs.plugins.ktlint.get().pluginId
```

Comment:
disable ktlint by not applying it to any of the project's modules and thus prevent the creation of ktlint gradle tasks. 

When we're ready to proceed we will uncomment out this 1 line and provide all the linting changes that come with it.

Reply:
Sounds good. I was about to ask how this would work since the tests were failing b/c of linting problems..